### PR TITLE
[Merged by Bors] - feat: miscellaneous Lie algebra lemmas

### PR DESCRIPTION
**Pull request by ocfnash**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by jcommelin — APPROVED** [review]
Thanks :tada: 

If CI passes, please merge this yourself, by adding a comment `bors r+`.

bors d+

**Comment by ocfnash** [comment]
bors merge

File: Mathlib/Algebra/Lie/Basic.lean [hunk 1]
```diff
@@ -261,6 +261,24 @@ instance : LieModule R L (M →ₗ[R] N)
...
+      map_smul' := by simp }
+  add_lie := fun x y m ↦ by ext n; simp [-neg_add_rev, neg_add]
+  lie_add := fun x m n ↦ by ext p; simp [-neg_add_rev, neg_add]
+  leibniz_lie := fun x m n ↦ by ext p; simp
+
+@[simp] lemma Module.Dual.lie_apply (f : M →ₗ[R] R) : ⁅x, f⁆ m = - f ⁅x, m⁆ := rfl
+
+instance Module.Dual.instLieModule : LieModule R L (M →ₗ[R] R) where
+  smul_lie := fun t x m ↦ by ext n; simp
+  lie_smul := fun t x m ↦ by ext n; simp
```
Comment (eric-wieser) [inline Mathlib/Algebra/Lie/Basic.lean:280]: nit: arguably more readable as
```suggestion
  lie_smul t x m := by ext n; simp
```
(Throughout)

Comment (eric-wieser) [reply Mathlib/Algebra/Lie/Basic.lean:280]: No need to cancel the merge though

Comment (ocfnash) [reply Mathlib/Algebra/Lie/Basic.lean:280]: Thanks, noted for the future.

**Pull request closed without merging**